README.md

```diff
@@ -20,8 +20,11 @@ heroku buildpacks:add --index 1 https://github.com/heroku/heroku-buildpack-apt
 
 #### Aptfile
 
+    # you can list packages
```

Comment:
This really needs to be in the readme, i was wondering what all those 3 lines where all about, i thought it was just one package being described in some weird way